client/my-sites/media-library/list.jsx

```diff
@@ -1,14 +1,15 @@
 /**
  * External dependencies
  */
-var ReactDom = require( 'react-dom' ),
-	React = require( 'react' ),
+var React = require( 'react' ),
 	clone = require( 'lodash/clone' ),
 	noop = require( 'lodash/noop' ),
 	filter = require( 'lodash/filter' ),
 	findIndex = require( 'lodash/findIndex' );
 
+import { AutoSizer, InfiniteLoader, Grid, WindowScroller } from 'react-virtualized';
 import { connect } from 'react-redux';
+import fill from 'lodash/fill';
```

Comment:
Sorry now I get the other comments... :) Didn't realise this only worked for Lodash.